### PR TITLE
add menu category for linux build of app

### PR DESCRIPTION
**Pull request by t3chguy**
for #3437

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -158,6 +158,7 @@
     ],
     "linux": {
       "target": "deb",
+      "category": "Network;InstantMessaging;P2P;",
```
Comment (dbkr) [inline package.json]: I'm not convinced Matrix is really p2p?

Comment (t3chguy) [inline package.json]: WebRTC :P
I'm just going off what @ara4n left in #3437

**Comment by ArchangeGabriel** [comment]
In ArchLinux I’m shipping Riot with the following line in the `.desktop` file:
`Categories=Network;InstantMessaging;Chat;IRCClient`

Obviously the first two we have in common are a good choice, whether adding Chat/IRCClient is a good idea, I’ll leave that up to you. I’ll adapt Arch `.desktop` to follow your choice.

**Comment by t3chguy** [comment]
Thanks for the heads up, Chat makes sense, IRCClient not quite so (to me anyway) even though you can access IRC through it. @dbkr what do you think :)?

**Pull request merged**